app/Amortization/Amortization.php

```diff
@@ -84,23 +84,17 @@ public function preview()
     {
         $IsSuperLoan = Str::contains($this->order->businessType->slug, 'super');
         $IsNoBs = Str::contains($this->order->businessType->slug, 'no_bs');
```

Comment:
I believe there is slug for  no bs renewal check the db

Reply:
ap_no_bs_renewal_non_verve, ap_no_bs_product_renewal_non_verve.
this is the challenge. i need to include the product too

Reply:
that is y i am doing that

Reply:
okay now